CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## NEXT_VERSION
```

Comment:
Lets make this part of the v0.3.0 release - I haven't quite cut that yet, but should be soon!

Reply:
Thanks for that callout! Will move that then merge.